gulp-tasks/test.js

```diff
@@ -772,6 +772,33 @@ function testContributors(filename, contents) {
   });
 }
 
+/**
+ * Tests and validates a _redirects.yaml file.
+ *   Note: The returned promise always resolves, it will never reject.
+ *
+ * @param {string} filename The name of the file to be tested.
+ * @return {Object} An object with all of the contributors.
+ */
+function testRedirects(filename, contents) {
+  return new Promise(function(resolve, reject) {
+    let msg;
+    let parsed = parseYAML(filename, contents);
+    let filepath = path.dirname(filename).split('/').splice(3).join('/');
+    filepath = path.join('/', 'web', filepath)
+    if (parsed.redirects && parsed.redirects.length > 0) {
+      parsed.redirects.forEach( item => {
```

Comment:
Nit: I *think* the google style for this is `.forEach((item) => {...` But I'm not positive.